src/elements/element.line.js

```diff
@@ -43,17 +40,17 @@ module.exports = Element.extend({
 		ctx.save();
 
 		// Stroke Line Options
-		ctx.lineCap = vm.borderCapStyle || globalOptionLineElements.borderCapStyle;
+		ctx.lineCap = valueOrDefault(vm.borderCapStyle, globalOptionLineElements.borderCapStyle);
 
 		// IE 9 and 10 do not support line dash
 		if (ctx.setLineDash) {
 			ctx.setLineDash(vm.borderDash || globalOptionLineElements.borderDash);
 		}
 
-		ctx.lineDashOffset = vm.borderDashOffset || globalOptionLineElements.borderDashOffset;
-		ctx.lineJoin = vm.borderJoinStyle || globalOptionLineElements.borderJoinStyle;
-		ctx.lineWidth = vm.borderWidth || globalOptionLineElements.borderWidth;
-		ctx.strokeStyle = vm.borderColor || globalDefaults.defaultColor;
+		ctx.lineDashOffset = valueOrDefault(vm.borderDashOffset, globalOptionLineElements.borderDashOffset);
+		ctx.lineJoin = valueOrDefault(vm.borderJoinStyle, globalOptionLineElements.borderJoinStyle);
+		ctx.lineWidth = valueOrDefault(vm.borderWidth, globalOptionLineElements.borderWidth);
+		ctx.strokeStyle = valueOrDefault(vm.borderColor, globalDefaults.defaultColor);
```

Comment:
Here I think we might need to have some helper that falls back multiple times: `valueOrDefault(vm.borderColor, globalOptionLineElements.borderColor, globalDefaults.defaultColor)`.

Reply:
`helpers.options.resolve` can be used here, but are multiple `? :` operators better as this is drawing method? 
```js
resolve([vm.borderColor, globalOptionLineElements.borderColor, globalDefaults.defaultColor]);
```

Reply:
Ideally those should be resolved before drawing, but for now I would leave it as is (for performance)

Reply:
It seems fine to me to merge this since it doesn't change the behavior at all. If we want to investigate moving some logic to the controller, I think that would best be handled separately